pkg/restore/actions/pod_volume_restore_action.go

```diff
@@ -157,15 +157,24 @@ func (a *PodVolumeRestoreAction) Execute(input *velero.RestoreItemActionExecuteI
 	runAsUser, runAsGroup, allowPrivilegeEscalation, secCtx := getSecurityContext(log, config)
 
 	var securityContext corev1.SecurityContext
-	if runAsUser == "" && runAsGroup == "" && allowPrivilegeEscalation == "" && secCtx == "" {
-		securityContext = defaultSecurityCtx()
-	} else {
+	securityContextSet := false
+	// Use securityContext settings from configmap if available
+	if runAsUser != "" || runAsGroup != "" || allowPrivilegeEscalation != "" || secCtx != "" {
 		securityContext, err = kube.ParseSecurityContext(runAsUser, runAsGroup, allowPrivilegeEscalation, secCtx)
 		if err != nil {
 			log.Errorf("Using default securityContext values, couldn't parse securityContext requirements: %s.", err)
-			securityContext = defaultSecurityCtx()
+		} else {
+			securityContextSet = true
 		}
 	}
+	// if first container in pod has a SecurityContext set, then copy this security context
```

Comment:
The log message printed in line 165 could be misleading, maybe just remove "Using default securityContext values" in the sentence?